dask-gateway-server/dask_gateway_server/utils.py

```diff
@@ -71,11 +71,10 @@ def __init__(self, context):
     async def __aenter__(self):
         if self.context.cancelled:
             raise asyncio.CancelledError
-        loop = get_running_loop()
         try:
-            self._task = asyncio.current_task(loop=loop)
+            self._task = asyncio.current_task()
```

Comment:
This seems to be new in 3.7: https://docs.python.org/3/library/asyncio-task.html#asyncio.current_task

dask-gateway-server currently requires python 3.6. I think we can safely update to requiring 3.7 (IIRC Dask is already doing that). I'll push a commit.

Reply:
Wait... you just removed `loop=loop`. So this might not have worked earlier with 3.6, or I'm confused :)

Reply:
This code already exists, I just dropped the loop argument. The `except` case handles the 3.6 case.